libraries/Microsoft.Bot.Builder.AI.LuisV3/LuisPredictionOptions.cs

```diff
@@ -0,0 +1,91 @@
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether all intents come back or only the top one.
+        /// </summary>
+        /// <value>
+        /// True for returning all intents.
+        /// </value>
+        public bool IncludeAllIntents { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not instance data should be included in response.
+        /// </summary>
+        /// <value>
+        /// A value indicating whether or not instance data should be included in response.
+        /// </value>
+        public bool IncludeInstanceData { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether API results should be included.
+        /// </summary>
+        /// <value>True to include API results.</value>
+        /// <remarks>This is mainly useful for testing or getting access to LUIS features not yet in the SDK.</remarks>
+        public bool IncludeAPIResults { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether queries should be logged in LUIS.
+        /// </summary>
+        /// <value>
+        /// If queries should be logged in LUIS in order to help build better models through active learning.
+        /// </value>
+        public bool Log { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets dynamic lists used to recognize entities for a particular query.
+        /// </summary>
+        /// <value>
+        /// Dynamic lists of things like contact names to recognize at query time.
+        /// </value>
+        public IList<DynamicList> DynamicLists { get; set; }
+
+        /// <summary>
+        /// Gets or sets external entities recognized in the query.
+        /// </summary>
+        /// <value>
+        /// External entities recognized in query.
+        /// </value>
+        public IList<ExternalEntity> ExternalEntities { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether external entities should override other means of recognizing entities.
+        /// </summary>
+        /// <value>
+        /// Boolean for if external entities should be preferred to the results from LUIS models.
+        /// </value>
+        public bool PreferExternalEntities { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether production (false) or staging (true) slot should be used.
+        /// </summary>
+        /// <value>
+        /// True for staging or false for production.
+        /// </value>
+        /// <remarks> 
+        /// LUIS supports both a production slot and a staging slot and this controls which is used.
+        /// If you specify a Version, then a private versioned slot is used instead.
+        /// </remarks>
+        public bool Staging { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets the specific version of the model to access.
```

Comment:
I think it'd be more suitable to make it like this.
```suggestion
        /// Gets or sets the specific version of the application to access.
```

Reply:
done